src/zenml/cli/stack_components.py

```diff
@@ -1351,12 +1362,165 @@ def flavor_group() -> None:
     )(explain_command)
 
 
+def register_annotator_subcommands() -> None:
+    """Registers CLI subcommands for the annotator."""
+    annotator_group = cli.commands.get("annotator")
+    if annotator_group:
+
+        @annotator_group.group(  # type: ignore[attr-defined]
+            cls=TagGroup,
+            help="Commands for interacting with annotation datasets.",
+        )
+        @click.pass_context
+        def dataset(ctx: click.Context) -> None:
+            """Interact with ZenML annotator datasets.
+
+            Args:
+                ctx: The click Context object.
+            """
+            repo = Repository()
+            active_stack = repo.zen_store.get_stack(name=repo.active_stack_name)
+            annotator_wrapper = active_stack.get_component_wrapper(
+                StackComponentType.ANNOTATOR
+            )
+            if annotator_wrapper is None:
+                cli_utils.error(
+                    "No active annotator found. Please register an annotator "
+                    "first and add it to your stack."
+                )
+                return
+
+            ctx.obj = annotator_wrapper.to_component()
+
+        @dataset.command(
+            "list",
+            context_settings={"ignore_unknown_options": True},
+            help="List the available datasets.",
+        )
+        @click.pass_obj
+        def dataset_list(annotator: "BaseAnnotator") -> None:
+            """List the available datasets.
+
+            Args:
+                annotator: The annotator stack component.
+            """
+            dataset_names = [
+                dataset.get_params()["title"]
+                for dataset in annotator.get_datasets()
+            ]
+            cli_utils.print_list_items(
+                list_items=dataset_names, column_title="DATASETS"
+            )
+
+        @dataset.command(
+            "stats", context_settings={"ignore_unknown_options": True}
+        )
+        @click.argument(
+            "name",
+            type=click.STRING,
+        )
+        @click.pass_obj
+        def dataset_stats(annotator: "BaseAnnotator", name: str) -> None:
+            """Display statistics about a dataset.
+
+            Args:
+                annotator: The annotator stack component.
+                name: The name of the dataset.
+            """
+            projects = annotator.get_datasets()
+            # TODO: make it impossible to create new datasets with same name
+            try:
+                project = list(
```

Comment:
Lots of LabelStudio specific code from here on

Reply:
Fixed now